azure-quantum/tests/integration/README.md

```diff
@@ -0,0 +1,14 @@
+# Integration tests
+
+These integration tests are for running end-to-end tests against a live service.
+To be able to run the tests, make sure to add a `config.ini` file to the package root (`azure-quantum`) containing your credential info in the following format:
```

Comment:
Since these config files contain plaintext credentials, it may be good to proactively add `config.ini` to `.gitignore` to prevent credential leakage. GitHub should automatically scan, but it can still be helpful.

Reply:
Sounds good

Reply:
Looks like it's already in there 👍